gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -88,12 +100,17 @@ <h3 class="mdc-layout-grid__cell--span-3">
                         {% set link = event | event_link %}
                         {% if link %}
                           <a href="{{ link }}"}>
+                        {% elif event | event_type == 'Introduced' and event | event_value == '0' %}
+                          <div class="tooltip">
```

Comment:
The <div> seems to overlap the <a>. Is that intentional?

Reply:
I'm not following? The opening and closing div tags are both in the equivalent `elif` branches as the anchor tags?

Reply:
Ah, disregard. I misinterpreted what was going on here. It seemed like this was 

```
<a>
<div>
</a>
</div>
```

But I now realise the <a> and <div> are mutually exclusive.